packages/builder/src/builderStore/store/frontend.js

```diff
@@ -523,7 +523,9 @@ export const getFrontendStore = () => {
         // Generate default props
         let props = { ...presetProps }
         settings.forEach(setting => {
-          if (setting.defaultValue !== undefined) {
+          if (setting.type === "multifield" && setting.selectAllFields) {
+            props[setting.key] = Object.keys(defaultDatasource.schema)
```

Comment:
We might want to change this to `Object.keys(defaultDatasource?.schema || {})` just since we can't guarantee a datasource was set, and this will throw an error otherwise.